src/sectionproperties/analysis/fea.py

```diff
@@ -92,24 +92,24 @@ def geometric_properties(
         ixy: float = 0.0
 
         # Gauss points for 6 point Gaussian integration
-        gps = gauss_points(n=6)
+        gps = gauss_points(n=4)
```

Comment:
For T6 (2nd order), 4-point quandrature (3rd order) is sufficient. 6-point quadrature (5th order) is an overkill. This saves 1/3 of time spent on integration.